pkg/vex/vex.go

```diff
@@ -105,10 +120,10 @@ func New() VEX {
 	}
 	return VEX{
 		Metadata: Metadata{
-			Context:    Context,
+			Context:    fmt.Sprintf("%s/%s", Context, SpecVersion),
```

Comment:
We probably want a first-class var or func for this `Context/SpecVersion` construction. It seems like we're using it in a few places